test/uk/gov/hmrc/economiccrimelevyaccount/base/SpecBase.scala

```diff
@@ -50,14 +55,25 @@ trait SpecBase
     with GuiceOneAppPerSuite
     with MockitoSugar
     with ScalaCheckPropertyChecks
-    with EclTestData {
+    with EclTestData
+    with BeforeAndAfterEach {
 
   val cc: ControllerComponents                         = stubControllerComponents()
   val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest()
   val appConfig: AppConfig                             = app.injector.instanceOf[AppConfig]
   val bodyParsers: PlayBodyParsers                     = app.injector.instanceOf[PlayBodyParsers]
+  val config: Config                                   = app.injector.instanceOf[Config]
+  val actorSystem: ActorSystem                         = ActorSystem("test")
   val fakeAuthorisedAction                             = new FakeAuthorisedAction(bodyParsers)
 
+  override def fakeApplication(): Application =
+    new GuiceApplicationBuilder()
+      .configure(
+        "metrics.jvm"     -> false,
```

Comment:
You can override retries here to just one and only 1ms, to prevent tests being overly long.

See example:

https://github.com/hmrc/economic-crime-levy-registration/blob/944f6bd1e935ec0ec1883d5684d6c4e4e760e06f/test/uk/gov/hmrc/economiccrimelevyregistration/connectors/DmsConnectorSpec.scala#L32